examples/powercli/eventbridge/README.md

```diff
@@ -0,0 +1,53 @@
+# AWS EventBridge Function
+
+## Description
+
+This function demonstrates using PowerShell and the EventBridge cmdlets to forward a vCenter Server Event to AWS EventBridge
+
+## Prerequisites
+
+* Already created custom [EventBridge Bus](https://docs.aws.amazon.com/eventbridge/latest/userguide/create-event-bus.html) and [EventBridge Rule](https://docs.aws.amazon.com/eventbridge/latest/userguide/create-rule-partner-events.html)
+* AWS Secret and Access Key with `AmazonEventBridgeFullAccess` policy
+
+## Instruction
+
+Step 1 - Initialize function, only required during the first deployment
+
+```
+faas-cli template pull
+```
+
+Step 2 - Update `stack.yml` and `eventbridge-secrets.json` with your environment information
```

Comment:
Perhaps advice to change IMAGE as well?

Reply:
I guess this depends if a user wants to just "consume" or "extend". My assumption was the former just to get started. Thoughts?

Reply:
Well, next step says „build the container“ so it would fail on push using your Docker account. I think build is not needed here (ie, for just fn consumption image can remain but step 3 needs to be removed then)